src/Middleware/SpaServices.Extensions/src/SpaOptions.cs

```diff
@@ -73,6 +73,6 @@ public PathString DefaultPage
         /// Gets or sets the maximum duration that a request will wait for the SPA
         /// to become ready to serve to the client.
         /// </summary>
-        public TimeSpan StartupTimeout { get; set; } = TimeSpan.FromSeconds(50);
+        public TimeSpan StartupTimeout { get; set; } = TimeSpan.FromSeconds(120);
```

Comment:
This is a product code change @SteveSandersonMS how do you feel about this?

Could we try this out in the tests first before moving to this?

Reply:
Seems like this is only used in development, so I'm fine with this.